draft-ietf-ccamp-actn-poi-pluggable-usecases-gaps.md

```diff
@@ -131,7 +131,7 @@ informative:
 
 --- abstract
 
-This document provides general overarching guidelines for control and management of packet over optical converged networks with coherent pluggables and focuses on operators' use cases and network scenarios. It provides a set of use cases which are needed for the control and management of the packet over optical networks which comprise devices with mixes of packet and optical functions where the optical functions may be provided on coherent pluggables. The document provides a gap analysis to solve the use cases.
+This document provides general overarching guidelines for control and management of packet over optical converged networks with programmable pluggables and focuses on operators' use cases and network scenarios. It provides a set of use cases which are needed for the control and management of the packet over optical networks which comprise devices with mixes of packet and optical functions where the optical functions may be provided on programmable pluggables. The document provides a gap analysis to solve the use cases.
```

Comment:
Should we briefly mentioned what we mean by "programmable" pluggables? Would at least mention this name has been chosen to cover both coherent and non-coherent pluggables which can be fully managed through an open and standard interface?

Reply:
That would help.  We can modify "with programmable pluggables" in "with programmable (either coherent and gray) pluggables